finality-aleph/src/session_map.rs

```diff
@@ -693,8 +651,8 @@ mod tests {
             );
         }
 
-        for block in blocks {
-            sender.unbounded_send(to_notification(block)).unwrap();
+        for n in PRUNING_THRESHOLD + 1..2 * PRUNING_THRESHOLD + 1 {
```

Comment:
I'm a bit nervous about this lacking parentheses. <_<"

Reply:
Kind of changed that